hack/deploy_traffic_flow_tests.sh

```diff
@@ -5,6 +5,14 @@ export KUBECONFIG=/root/kubeconfig.ocpcluster
 nodes=$(oc get nodes)
 export worker=$(echo "$nodes" | grep -oP '^worker-[^\s]*')
 
+
+# wa for https://issues.redhat.com/browse/IIC-364
+make undeploy
+podman push $(hostname):5000/intel_vsp:dev # Intel vsp is not multiarch, make sure intel_vsp:dev is x86_64
```

Comment:
How does this ensure that the image is x86_64 by pushing it again?

Reply:
We still can't cross compile Intel's VSP with podman, so we are building it on both sides when deploying the dpu operator. The images will have the same name currently, so this makes sure the x86 one is most recent.

Now that I am saying this, I should probably just push a patch to CDA to push these images as a buildah manifest so we don't need to worry about this

Reply:
thanks

Reply:
Added in https://github.com/bn222/cluster-deployment-automation/pull/258, when this merges I will add an additional bump to CDA in this PR and remove this WA

Reply:
resolved